### PR TITLE
Fix getting path to apkanalyzer

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Addresses https://github.com/appium/appium/issues/10120#issuecomment-364227397

### CODE REVIEW

File: lib/tools/android-manifest.js [hunk 1]
```diff
@@ -159,7 +159,10 @@ async function extractApkInfoWithApkanalyzer (localApk) {
  */
 manifestMethods.packageAndLaunchActivityFromManifest = async function (localApk) {
   const apkInfoGetters = [
-    async () => await extractApkInfoWithApkanalyzer(localApk),
+    async () => {
+      const apkanalyzerPath = await getApkanalyzerForOs(this);
```
Comment (imurchie) [inline lib/tools/android-manifest.js]: Any reason to do this here rather than in the `extractApkInfoWithApkanalyzer` function?

Comment (mykola-mokhnach) [reply lib/tools/android-manifest.js]: we need `this` context

Comment (imurchie) [reply lib/tools/android-manifest.js]: Ah! Makes sense.

**Comment by imurchie** [comment]
Published in `appium-adb@6.2.1`

**Pull request merged**